src/editors/mongoDBDocumentService.ts

```diff
@@ -90,8 +94,8 @@ export default class MongoDBDocumentService {
       );
       await findOneAndReplace(
         namespace,
-        { _id: documentId },
-        newDocument as Document,
+        { _id: documentId as ObjectId },
```

Comment:
Do we need to cast this to ObjectId here? Maybe there's a better type like `Document` or `any`, as _id can be types besides `ObjectId`.
```suggestion
        { _id: documentId as ObjectId },
```

Reply:
Maybe any would be better 👍 Document doesn't seem fit for string values.

Reply:
Yeah scratch that `Document` suggestion, it's not the right type here. any is fine by me 👍